### PR TITLE
Python chapter1 (palindrome_permutation)

### PR DESCRIPTION
**Pull request by CheRayLiu**
Added Palindrome Permutation and its tests

### CODE REVIEW

File: python/questions/chapter1.py [hunk 1]
```diff
@@ -27,4 +27,22 @@ def is_permutation(string_a, string_b):
...
+# Space: O(n)
+def palindrome_permutation(string):
+    char_dict = {}
+    for char in string:
+        if char not in char_dict:
+            char_dict[char] = 1
+        else:
+            char_dict[char] += 1
+    odd_count = 0
+    for char,count in char_dict.items():
```
Comment (KevinShiCA) [inline python/questions/chapter1.py]: You can use `char_dict.values()`

File: python/questions/chapter1.py [hunk 2]
```diff
@@ -27,4 +27,22 @@ def is_permutation(string_a, string_b):
...
+        if count%2 == 1:
+            odd_count += 1
+    return True if odd_count in {0,1} else False
```
Comment (KevinShiCA) [inline python/questions/chapter1.py]: `return odd_count in {0, 1}`

File: python/questions/chapter1.py (hunk 1 above)
Comment (CheRayLiu) [reply python/questions/chapter1.py]: Very true

File: python/questions/chapter1.py (hunk 2 above)
Comment (CheRayLiu) [reply python/questions/chapter1.py]: Smart

**Review by KevinShiCA — APPROVED** [review]

**Pull request merged**